### ISSUE TITLE
Not working for J-Series

### ISSUE THREAD

**Issue by Jey-Cee** [andrei10k/new-samsung-remote#1]
Hi,

i testet your modul to control my J-Series TV, but it want work.
Response from sendkey is undefined. There is no error or something else.

`const Remote = require('new-samsung-remote');
        
const config = {
        ip_address: '192.168.0.234', // required: IP address of your Samsung Smart TV 
        name: "New Samsung Remote"
};      
    
const remote = Remote(config);
    
remote.api_active();
    
remote.sendKey('KEY_MUTE', function(err, res){
    if (err) {
        throw new Error(err);
    } else {
        console.log(res);
    }
});
`

**Comment by andrei10k** [comment]
Are you positive that 192.168.0.234 is the IP of your tv set and that you are on the same network? Also your tv must be 2015-2016 version because they changed the protocol for newer devices. To be honest I tested it on K series. I'll test it on J series later today and let you know.

**Comment by Jey-Cee** [comment]
Yes it is the right ip and it is on the same network, i can retrive information from the tv and is found by upnp devices. 
What i know is that samsung has changed the os from android to tizen os. 
Thank you for now. I will wait for your answer.

**Comment by andrei10k** [comment]
From what I saw your model was produced in 2015. Maybe they still used the old protocols back then. Try this module instead: https://github.com/natalan/samsung-remote. At first I tried using that myself, until I realized it no longer works for newer models. Maybe it will still work for you. Let me know. Cheers

**Comment by Jey-Cee** [comment]
Nope, also not working. Damn stupid product strategy by samsung.

**Comment by schneiderr** [comment]
I also have a J series TV which gives me the same undefined response. (UN55JS7000). I am able to grab the tv information settings at xxx.xxx.xxx.xxxx:8001/api/v2. 

I've also tried the older model with no luck.

**Comment by nspinelli** [comment]
Has anyone gotten this to work? I am testing on UN60JS6900 and am getting the following response:
{"event":"ms.error","data":{"message":"unrecognized method value : ms.remote.control"}}

**Comment by nspinelli** [comment]
@andrei10k How did you find out what method to use when posting your CMD? 
"method": "ms.remote.control"
Are you aware of any other "Methods"?

**Comment by RTK** [comment]
I can confirm that this module does not work with Samsung J series, even thought the request to http://ip/api/v2/ delivers a valid response, thus i assume there is a remote api. Unfortunately I could not find any information for this and gave up crawling into Samsung's poorly documented API references. Does anyone have any useful hints? Like... a documentation? 😮

**Comment by andrei10k** [comment]
@RTK you are absolutely right. The documentation is garbage. I read all I could find on their website for another project. I even contacted some of their engineers. You can find them on their github account. Maybe you should follow that track and if you find anything useful please share it with me so I can update the module. Good luck!
